packages/tests-postman/data/service-providers.json

```diff
@@ -42,6 +43,7 @@
         "oauth2": {
             "access_token_url": "https://transmute-industries.auth0.com/oauth/token",
             "client_id": "CGACwnX3KuIk9bfNgb7pSAd9SZxdCAEl",
+            "client_secret_envvar": "CLIENT_SECRET_b0643936e30843b622c78ad353da19123e4583817491fe05e2e9d39824983bee",
```

Comment:
probably a good idea to make this a named secret too?
rather than have it be a long string that will make secrets management super hard

Reply:
This can be changed to anything you like - I left it set to the current secret value to prevent BC break.

Reply:
I created #77 for this

Reply:
changed to approved now that issue is created